tests/adiabatic_conditions_composition_function/depth_average.gnuplot

```diff
@@ -5,73 +5,73 @@
 # For a description of the GNUPLOT format see the GNUPLOT manual.
 #
 # <depth> <time> <temperature> <C_1> <adiabatic_temperature> <adiabatic_pressure> <adiabatic_density> <adiabatic_density_derivative> <velocity_magnitude> <sinking_velocity> <rising_velocity> <viscosity> <log_viscosity> <vertical_heat_flux> <vertical_mass_flux> <C_1_mass> 
-0 0 1613 0 1613 0 3300 0 7.06667e-16 7.80424e-17 7.80424e-17 1e+21 21 3.92899e-08 4.95593e-13 0 
-289000 0 1613 0 1613 0 3300 0 7.06667e-16 7.80424e-17 7.80424e-17 1e+21 21 3.92899e-08 4.95593e-13 0 
+0 0 1613 0 1613 4.64175e+09 3300 0 7.06667e-16 7.21373e-17 7.80424e-17 1e+21 21 3.92907e-08 4.95593e-13 0 
+289000 0 1613 0 1613 4.64175e+09 3300 0 7.06667e-16 7.21373e-17 7.80424e-17 1e+21 21 3.92907e-08 4.95593e-13 0 
```

Comment:
This file has changes that are not in the adiabatic properties, but instead in the sinking velocity and the vertical heat flux. They are very small, but I don't see why anything should be changed at all. Can you check what happened here?